installer/deb/scripts/postinst

```diff
@@ -73,36 +70,12 @@ set_local_dns() {
     echo "DNS configuration complete."
 }
 
-# Download and install mkcert, generate a certificate for MassaStation.
-generate_certificate() {
-    curl -sL $MKCERT_URL -o mkcert || fatal "mkcert download failed."
-    chmod +x mkcert || fatal "mkcert permission change failed."
-
-    # We ensure that CAROOT is set. Otherwise, mkcert installation will fail.
-    # It especially happens on systems that install packages in sandboxed environments.
-    export CAROOT=$(./mkcert -CAROOT)
-    if [[ -z $CAROOT ]]; then
-        CAROOT=/usr/local/share/ca-certificates/
-    fi
-
-    ./mkcert -install || fatal "mkcert installation failed."
-    mkdir -m 777 -p $MASSASTATION_CERT_DIR || fatal "certs directory creation failed."
-    ./mkcert -cert-file $MASSASTATION_CERT_DIR/cert.pem -key-file $MASSASTATION_CERT_DIR/cert-key.pem station.massa || fatal "mkcert certificate generation failed."
-    chmod 644 $MASSASTATION_CERT_DIR/cert.pem $MASSASTATION_CERT_DIR/cert-key.pem || fatal "certs permission change failed."
-
-    rm mkcert || fatal "mkcert removal failed."
-}
-
```

Comment:
Didn't we said that we would keep this until we are 100% sure everything is working as expected ? 🤔

Reply:
we moved to plan B ^^